docs/getting-started-monitoring.md

```diff
@@ -118,6 +123,16 @@ First, you need to install the following tools used in this guide.
    helm install scalar-monitoring prometheus-community/kube-prometheus-stack -n monitoring -f scalar-prometheus-custom-values.yaml
    ```
 
+1. Add the `grafana` helm repository.
+  ```console
+  helm repo add grafana https://grafana.github.io/helm-charts
+  ```
+
+1. Deploy the `loki-stack`.
+   ```console
+   helm install loki-stack grafana/loki-stack -n monitoring -f scalar-loki-stack-custom-values.yaml
```

Comment:
```suggestion
   helm install scalar-logging-loki grafana/loki-stack -n monitoring -f scalar-loki-stack-custom-values.yaml
```
I think `loki-stack` is product name, and users can set arbitrary name as `release name` of helm.
And, I think it is better to use more understandable `release name` that users can know the purpose of deployment is Logging for Scalar products.
(For example, I use `scalar-monitoring` for release name of `kube-prometheus-stack` in this document.)
So, I think `scalar-logging-loki` is better as `release name` in the sample.

---

Reference information

First, I thought it is better to use `scalar-xxxx` (`scalar-logging`) as `release name` for consistent with `scalar-monitoring` in this doc.
However, when I set `scalar-logging` as `release name` like the following, the error occurs and promtail cannot send logs to Loki Server.

* command
  ```
  helm install scalar-logging grafana/loki-stack -n monitoring -f scalar-loki-stack-custom-values.yaml
  ```
* error
  ```
  level=warn ts=2022-05-09T05:58:47.984339253Z caller=client.go:349 component=client host=scalar-logging:3100 msg="error sending batch, will retry" status=-1 error="Post \"http://scalar-logging:3100/loki/api/v1/push\": dial tcp: lookup scalar-logging on 10.96.0.10:53: no such host"
  ```

The cause of the above error is mismatch of `Service resource name` of Loki server and `URL configuration` of promtail.

In the above deployment, the service resource name of Loki server is `scalar-logging-loki`.

```
$ kubectl -n monitoring get svc
NAME                           TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
scalar-logging-loki            ClusterIP   10.108.35.62   <none>        3100/TCP   5m22s
scalar-logging-loki-headless   ClusterIP   None           <none>        3100/TCP   5m22s
```
So, promtail should send logs to `http://scalar-logging-loki:3100`.
However, in the promtail configuration, it try to send logs to `http://scalar-logging:3100`.
This URL mismatch causes the error.

Also, this mismatch is caused by automatically naming in the Loki chart.
When there is no string `loki` in the `release name`, loki's helm charts add suffix `-loki` automatically.

* https://github.com/grafana/helm-charts/blob/loki-2.11.1/charts/loki/templates/_helpers.tpl#L14-L22

And, that name (with suffix) will be used as name of `Service` resource of Loki server.

* https://github.com/grafana/helm-charts/blob/loki-2.11.1/charts/loki/templates/service.yaml#L4

However, the promtail use `release name` without suffix `-loki` in the URL to send logs to Loki server by default.

* https://github.com/grafana/helm-charts/blob/loki-stack-2.6.4/charts/loki-stack/values.yaml#L8

The above mismatch causes some errors like the following.

* Example
  * `helm install scalar-logging ...`
    * Service resource name of Loki server is `scalar-logging-loki`.
      * The suffix `-loki` is added automatically.
    * Promtail use URL `http://scalar-logging:3100` to send logs to Loki server.
    * There is mismatch.
  * `helm install scalar-logging-loki ...`
    * Service resource name of Loki server is `scalar-logging-loki`.
      * The `release name` includes string `loki`, so the suffix `-loki` is not added.
    * Promtail use URL `http://scalar-logging-loki:3100` to send logs to Loki server.
    * There is NO mismatch.
  * `helm install my-project-test ...`
    * Service resource name of Loki server is `my-project-test-loki`.
      * The suffix `-loki` is added automatically.
    * Promtail use URL `http://my-project-test:3100` to send logs to Loki server.
    * There is mismatch.
  * `helm install my-project-loki-test ...`
    * Service resource name of Loki server is `my-project-loki-test`.
      * The `release name` includes string `loki`, so the suffix `-loki` is not added.
    * Promtail use URL `http://my-project-loki-test:3100` to send logs to Loki server.
    * There is NO mismatch.

So, I think the one of simple way to avoid this error is including string `loki` in the `release name` of helm, like `scalar-logging-loki`.

If we want to use `release-name` that does not include string `loki`, it seems that we can avoid the error by additional configuration of promtail like the following.  

* scalar-loki-stack-custom-values.yaml
```yaml
promtail:
  config:
    lokiAddress: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
    snippets:
      scrapeConfigs: |
        - job_name: scalardl-ledger
...
```

This configuration set URL to send logs from promtail to Loki server.
However, I didn't check/test another feature than sending logs. (There is possibility that another release name mismatch occurs.)
So, I think it is better to make `release name` of loki and promtail are matching, like `scalar-logging-loki` in the sample command.

Reply:
Thanks for the advice.
I revised it in https://github.com/scalar-labs/helm-charts/pull/87/commits/eb349d4503420bc0cdcf55ac9bef9de68a97ea80